### ISSUE TITLE
http://[...]/jobs/X endpoint doesn't have error information

### ISSUE THREAD

**Issue by seamusabshere** [faradayio/falconeri#28]
it shows status=error but doesn't provide any error details

**Comment by emk** [comment]
Yes, this is a good thing to tackle. See #32 for the tracking issue.

**Comment by emk** [comment]
Actually, now that we have the `babysitter.rs` code, we also have a case where a job can fail _without_ any individual datum failures (for example, a zombie job). So we need to add job-level error handling, too.
